src/Button.js

```diff
@@ -40,6 +40,8 @@ const Button = styled.button`
   text-align: center;
   text-decoration: none;
   font-weight: 600;
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
```

Comment:
This was covered in #112 – can you remove these lines and merge the latest master into this branch?

Reply:
`text-rendering: optimizeLegibility;` isn't necessary?